coordinator.py

```diff
@@ -94,10 +126,12 @@ def run(self, X, i_to_addr, i_to_sk, malicious):
                 send_count += len(data)
 
                 logging.debug(f'Enough participants are ready, starting new session with sid {self.model.sid_ctr}')
+                T = [i for _, i in data]
```

Comment:
This is a bit ugly, but open to other suggestions

Reply:
I gave it a try, not entirely sure if mine is better. It makes the inspection of the model more explicit, which I think is the controversial thing here: simply stealing state from the model is ugly from a programming point of view but it keeps the model itself clean from the attacker handling, and so the model stays close the pseudocode in the paper.

Anyway, it's just a prototype..